tests/dir.props

```diff
@@ -114,6 +114,11 @@
     <PackageVersionArg>/p:CoreClrPackageVersion=$(VersionToRestore)</PackageVersionArg>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <RuntimeIdentifiersOverwrite Condition="'$(__RuntimeId)' != ''">$(__RuntimeId)</RuntimeIdentifiersOverwrite>
+    <RuntimeIdentifiersOverwrite Condition="'$(__RuntimeId)' == ''">win-$(__BuildArch)</RuntimeIdentifiersOverwrite>
```

Comment:
Are we sure this is the correct fallback? Can we not share the same logic as TargetRID here so that we don't compute these slightly different in different places?